ai_diffusion/style.py

```diff
@@ -40,6 +40,35 @@ class StyleSettings:
         " [ComfyUI]/models/checkpoints.",
     )
 
+    free_u = Setting(
+        "Use FreeU",
+        False,
+        "FreeU is a method of modifying the UNet that can improve model output quality, "
+        "without any additional speed, memory, or computational costs. However the resulting "
+        "images may suffer from oversaturation.",
+    )
+
+    b1 = Setting(
```

Comment:
If we want to keep them in the settings file, can these be grouped? Maybe put a dict here
```
free_u_params = Setting("FreeU Parameters", dict(b1=1.1, b2=1.2, s1=0.6, s2=0.4))
```
Make a copy of the defaults  in `Style.__init__` in that case.